### DISCUSSION TITLE
Server Error from content-styles - Upgraded from 21.1.6 to 21.6.0

### DISCUSSION THREAD

**Discussion by bewards** [Sitecore/jss#1750] [Help] [answered]
### Description

After upgrading to 21.6.0 (nextjs, nextjs-sxa), I see the addition of the content-styles.ts page-props plugin. Are there any necessary Sitecore templates to get this working after the upgrade?

![image](https://github.com/Sitecore/jss/assets/3927169/e2987fbc-1f2b-4f98-b9af-63f1362f76a5)

### Additional information

- Found on Local Docker after upgrading
- If I disable the content-styles plugin, the application works again

**Comment by art-alexeyenko** [comment, marked as answer]
@bewards that file (content-styles.ts) should be removed for SXP deployments, it's an xmcloud only functionality. I see our upgrade guide missing this point atm - it will be updated.
Thank you for brining our attention to this.

**Reply by art-alexeyenko** [reply]
The guide was updated here:
https://doc.sitecore.com/xp/en/developers/hd/21/sitecore-headless-development/upgrade-jss-21-5-next-js-apps-to-version-21-6.html
